ansible/roles/airflow/templates/airflow.cfg.j2

```diff
@@ -0,0 +1,67 @@
+[core]
+# The home folder for airflow, default is ~/airflow
+airflow_home = {{ airflow_home }}
+
+# The folder where your airflow pipelines live, most likely a
+# subfolder in a code repository
+# This path must be absolute
+dags_folder = {{ airflow_home }}/dags
+
+# The folder where airflow should store its log files
+# This path must be absolute
+base_log_folder = {{ airflow_home }}/logs
+
+# The executor class that airflow should use. Choices include
+# SequentialExecutor, LocalExecutor, CeleryExecutor
+executor = LocalExecutor
```

Comment:
docs recommend `CeleryExecutor` for production usage, do we have plans to move to that later?

Reply:
As we scale, we probably. For now it does not seem worth it as all the tasks can probably be run locally, especially since the db machine does all the heavy lifting since most of the work is raw sql, and the added complexity of celery does not buy us much unless we have other workers.